superset/assets/visualizations/chord.css

```diff
@@ -0,0 +1,17 @@
+.chord svg #circle circle {
```

Comment:
by convention each visualization has its own `css` file, and the entries need to be "namespaced" properly to make sure that one viz's css doesn't spill and apply to other viz types